stock_landed_costs_average/model/stock_landed_costs.py

```diff
@@ -33,7 +33,7 @@ class StockLandedCost(models.Model):
     @api.onchange('invoice_ids')
     def onchange_invoice_ids(self):
         for lc_brw in self:
-            lc_brw.update({'cost_lines': [(6, False, {})]})
+            lc_brw.cost_lines.unlink()
```

Comment:
@osvalr I have to check the use of this unlink here,
I had some problems before that is the reason of using update with (6, False {})